docs/src/types.md

```diff
@@ -0,0 +1,128 @@
+# Types and methods
+
+## Types and their constructors
+
+None of the types below need to be constructed directly; they arise from
+performing operations between `LinearMap` objects or by calling the `LinearMap`
+constructor described next.
+
+### `LinearMap`
+
+Abstract supertype
+
+```@docs
+LinearMaps.LinearMap
+```
+
+### `FunctionMap`
+
+Type for wrapping an arbitrary function that is supposed to implement the
+matrix-vector product as a `LinearMap`; see above.
+
+### `WrappedMap`
+
+Type for wrapping an `AbstractMatrix` or `LinearMap` and to possible redefine
+the properties `isreal`, `issymmetric`, `ishermitian` and `isposdef`. An
+`AbstractMatrix` will automatically be converted to a `WrappedMap` when it is
+combined with other `AbstractLinearMap` objects via linear combination or
```

Comment:
There is no `AbstractLinearMap`, should just be `LinearMap`

Reply:
This must be from a very old commit. 😃